portfolio/src/main/webapp/script.js

```diff
@@ -32,10 +32,17 @@ function changeCatPhoto() {
 }
 
 /* Displays all comments left on page. */
-async function getComments() {
-  const response = await fetch('/data').then(response => response.json())
+async function getComments(max) {
+  document.getElementById('comments-section').innerHTML = "";
+  const response = await fetch('/data?max-comments=' + max).then(response => response.json())
   for (const comment of response) {
       document.getElementById('comments-section').innerHTML +=
-        comment['commentText'] + " --" + comment['displayName'] + "<br>";
+        comment["commentText"] + " --" + comment["displayName"] + "<br>";
```

Comment:
Is there a reason this changed to using double quotes? In JS, string literals should use single quotes.

https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes